src/api/app/assets/stylesheets/webui/application/image_templates.css.sass

```diff
@@ -2,6 +2,23 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+.break-words
+  /* These are technically the same, but use both */
+  overflow-wrap: break-word
+  word-wrap: break-word
+
+  /* Warning: Needed for oldIE support, but words are broken up letter-by-letter */
+   -ms-word-break: break-all
+   word-break: break-all
+
+   /* Non standard for webkit */
+   word-break: break-word
```

Comment:
only if the browser understands it - i.e. webkit. All other browsers will ignore it as syntax error.